File: 1818/1818.go
func minimumAbsoluteSumDifference(root *TreeNode) int {
	if root == nil {
		return 0
	}
	l := minimumAbsoluteSumDifference(root.Left)
	r := minimumAbsoluteSumDifference(root.Right)
	if l > r {
		return 1 + l
	}
	return 1 + r
}

File: 1818/1818.c
int minimumAbsoluteSumDifference(struct TreeNode* root) {
    if (!root) return 0;
    int l = minimumAbsoluteSumDifference(root->left);
    int r = minimumAbsoluteSumDifference(root->right);
    return 1 + (l > r ? l : r);
}